post-processing-service/src/main/java/gov/cdc/etldatapipeline/postprocessingservice/service/PostProcessingService.java

```diff
@@ -247,12 +248,14 @@ protected void processCachedIds() {
                     case CASE_MANAGEMENT:
                         processTopic(keyTopic, entity, ids,
                                 investigationRepository::executeStoredProcForCaseManagement);
+                        processTopic(keyTopic, entity.getName(), ids,
+                                investigationRepository::executeStoredProcForFStdPageCase, "sp_f_std_page_case_postprocessing");
                         break;
                     case INTERVIEW:
                         processTopic(keyTopic, entity, ids,
                                 postProcRepository::executeStoredProcForDInterview);
-                        processTopic(keyTopic, entity.getName(), ids,
-                                postProcRepository::executeStoredProcForFInterviewCase, "sp_f_interview_case_postprocessing");
+                        processTopic(keyTopic, Entity.F_STD_PAGE_CASE, ids,
+                                postProcRepository::executeStoredProcForFInterviewCase);
```

Comment:
Why is F_STD_PAGE_CASE the entity for this?

Reply:
It has all necessary info for logging and sp name to call.